CHANGELOG.md

```
@@ -2,6 +2,7 @@
 
 ### 10.2.8 - SNAPSHOT
   * Feat: Enhanced the `MxParseUtils` to be lenient when the XML declaration has empty or invalid version attribute
+  * (PW-) Block-5-5PDE-is-getting-generated-even-though-PssblDplct-value-is-false in MX AppHdr. CpyDplct is excluded for PDE check.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)